### ISSUE TITLE
You are not allowed to stop disconnection

### ISSUE THREAD

**Issue by MyTheValentinus** [LibrIT/passhport#397]
**Describe the bug**
When i exit a ssh connection and return to main "menu", press CTRL+C to exit i have a message per previously SSH sessions "You are not allowed to stop disconnection".

**To Reproduce**
Connect to passhport server, connect to one or multiple connections, press CTRL C after you exit connection.

**Comment by MyTheValentinus** [comment]
Id due to this: https://github.com/LibrIT/passhport/blob/master/passhport/passhport-connect.sh#L23

**Comment by elg** [comment]
Yes. Ctrl-C IS not allowed for security reasons. You have to use standard 15 signal ( Ctrl-D ).
Can you confirm it answer tour expectations?

**Comment by MyTheValentinus** [comment]
Humm, yeah ok `CTRL+D` is good ! Add this to doc ;) 
All sysadmin i know are "barbarians" and press `CTRL+C` ^^

Thanks !

**Comment by elg** [comment]
For details : if, with some Luck/calculation you sens the signal 9 during the right timing, you can obtain a local connection to the passhport bastion ans escalade to all targets. This simple Trap avoid this.

**Comment by elg** [comment]
Doc will not be read. We should s change the error message to lead people to Ctrl-D. Right.

**Comment by MyTheValentinus** [comment]
Yeah, i just inspect that for the trap ! 
Write this in the doc, sys admin love details about the softwate work. And yeah change message would be more user friendly ^^
Thanks !
